### ISSUE TITLE
[Accessibility] "Collapse panes" buttons should have different labels

### ISSUE THREAD

**Issue by fvsch** [devtools-html/debugger.html#7741]
We have two buttons with the same "Collapse panes" label:

- One collapses the sources navigation
- One collapses the right-hand panes

This creates an accessibility issue because both buttons appear in the same context, and without access to visual information (proximity of the button with either sidebar) you can't differentiate them.

I suggest using a different label for the first button, maybe "Collapse navigation"?

/cc @yzen 

Related issue: #4073.

**Comment by fvsch** [comment]
@Matt Croud In the Debugger, the 2 sidebar-collapsing buttons have the same label, "Collapse panes".
Could we come up with 2 different labels that make sense, for accessibility's sake?

- "Collapse sources" probably works for the left sidebar. "Collapse sources and outline" may work too.
- "Collapse breakpoints" for the right sidebar maybe? (It's a approximation but 70% of the right sidebar content is different types of breakpoints so it may work.)

Any ideas?

**Comment by fvsch** [comment]
I've settled on:

- "Collapse Sources and Outline panes" / "Expand Sources and Outline panes"
- "Collapse Breakpoints pane" / "Expand Breakpoints pane"

**Comment by jasonLaster** [comment]
Hmm, this is a bit tough ... I think of the first pane as navigation (source, functions) and the second pane for pausing (expressions, bps, stacks, scopes), but it is a bit of a bucket so tough to describe...
